tensorflow/stream_executor/cuda/cuda_blas.cc

```diff
@@ -413,10 +402,10 @@ bool CUDABlas::DoBlasInternalImpl(FuncT cublas_func, Stream *stream,
                                            : CUBLAS_POINTER_MODE_DEVICE)) {
     return false;
   }
-#if CUDA_VERSION >= 9000
+#if CUBLAS_VER_MAJOR >= 11
   ScopedCublasMathMode math_mode{blas_};
-  if (use_tensor_op_math) {
-    if (!math_mode.Init(CUBLAS_TENSOR_OP_MATH)) {
+  if (!tensorflow::tf32_execution_allowed()) {
+    if (!math_mode.Init(CUBLAS_DEFAULT_MATH)) {
```

Comment:
I don't think this is necessary. The default is `CUBLAS_DEFAULT_MATH` and we only set this otherwise if TF32 is allowed.

Reply:
Good catch. Fixed.